README.md

```diff
@@ -63,1498 +104,468 @@ Delegating a capability is like the statement `message`. Task is akin to `messag
-  fx      Effects
+## Type Tag
+[Type Tag]: #type-tag
  
-  # All the other metadata
-  meta    {String : Any}
-
-  # Principal that issued this receipt. If omitted issuer is
-  # inferred from the invocation task audience.
-  iss     optional Principal
-
-  # When issuer is different from executor this MUST hold a UCAN
-  # delegation chain from executor to the issuer. This should be 
-  # omitted when the executor is the issuer.
-  prf     [&UCAN]
-
-  # Signature from the "iss".
-  s       Varsig
-}
-
-type Result union {
-  | any    "ok"    # Success
-  | any    "error" # Error
-} representation kinded
-
-# Represents a request to invoke enclosed set of tasks concurrently
-type Effects struct {
-  # Primary set of tasks to be invoked
-  fork      [&Task]
-  
-  # Continuation for straight-line programs
-  join       optional &Task
-}
-
-# Way to reference result of the Task
-type Await union {
-  | &Task    "await/*"
-  | &Task    "await/ok"
-  | &Task    "await/error"
-} representation keyed
-```
+The UCAN envelope's payload tag MUST be `ucan/inv@1.0.0-rc.1`.
 
-# 3 Task
+## Invocation Payload
+[Invocation Payload]: #invocation-payload
 
-A Task is the smallest unit of work that can be requested from a UCAN. It describes one `(resource, ability, input)` triple. The `input` field is free form, and depend on the specific resource and ability being interacted with, and is not described in this specification.
+The Invocation Payload attaches sender, receiver, and provenance to the [Task].
+ 
+| Field   | Type               | Required | Description                                                        |
+|---------|--------------------|----------|--------------------------------------------------------------------|
+| `iss`   | `DID`              | Yes      | The DID of the [Invoker]                                           |
+| `sub`   | `DID`              | Yes      | The [Subject] being invoked                                        |
+| `aud`   | `DID`              | No       | The DID of the intended [Executor] if different from the [Subject] |
+| `cmd`   | `String`           | Yes      | The [Command]                                                      |
+| `args`  | `{String : Any}`   | Yes      | The [Command]'s [Arguments]                                        |
+| `prf`   | `[&Delegation]`    | Yes      | [Delegation]s that prove the chain of authority                    |
+| `meta`  | `{String : Any}`   | No       | Arbitrary [Metadata]                                               |
+| `nonce` | `Bytes`            | No       | A unique, random nonce                                             |
+| `exp`   | `Integer`[^js-num] | No       | The timestamp at which the Invocation becomes invalid              |
+| `iat`   | `Integer`[^js-num] | No       | The timestamp at which the Invocation was created                  |
+| `cause` | `&Receipt`         | No       | An OPTIONAL CID of the [Receipt] that enqueued the [Task]          |
```

Comment:
[Receipt] is also not defined anywhere... self-explanatory perhaps, but a place-holder section with a one-sentence summary would be good for now?